networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,22 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+[id="network-observability-operator-release-notes-1-4-1"]
+== Network Observability Operator 1.4.1
+The following advisory is available for the Network Observability Operator 1.4.1:
+
+* link:https://access.redhat.com/errata/RHSA-2023:122114[2023:122114 Network Observability Operator 1.4.1]
+
+=== CVEs
+* link:https://access.redhat.com/security/cve/cve-2023-44487[2023-44487]
+* link:https://access.redhat.com/security/cve/cve-2023-39325[2023-39325]
```

Comment:
I've added more CVE fixes to the list, this morning:
- [CVE-2023-29406](https://access.redhat.com/security/cve/CVE-2023-29406)
- [CVE-2023-29409](https://access.redhat.com/security/cve/CVE-2023-29409)
- [CVE-2023-39318](https://access.redhat.com/security/cve/CVE-2023-39318)
- [CVE-2023-39319](https://access.redhat.com/security/cve/CVE-2023-39319)
- [CVE-2023-39322](https://access.redhat.com/security/cve/CVE-2023-39322)